src/hooks/useUpcomingReleases.tsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from 'react';
+import { useReleaseData } from './useReleaseHistory';
+
+export type ReleaseTypes = 'current' | 'lts' | 'maintenance' | 'endoflife';
+
+export interface UpcomingReleaseData {
+  releaseDate: string;
+  releaseType: ReleaseTypes;
+  alreadyReleased: boolean;
+}
+
+export interface UpcomingRelease {
+  title: string;
+  releases: UpcomingReleaseData[];
+}
+
+export const RELEASE_TYPES: { [key in ReleaseTypes]: string } = {
```

Comment:
What do you think about making this enum also?

Reply:
This was also previously implemented like this 😅